src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -129,12 +129,12 @@ function useChatUtilsHook(): UseChatUtilsHook {
     setChannelAsRead(channel);
     if (channel?.channelType === ANON_PM) {
       if (from === GROUP_INFO) {
-        return openChat('SampleChatScreen');
+        return openChat('SampleChatScreen', 'AnonymousChannelList');
       }
       navigation.navigate('SampleChatScreen');
     } else {
       if (from === GROUP_INFO) {
-        return openChat('SignedChatScreen');
+        return openChat('SignedChatScreen', 'SignedChannelList');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `openChat` function is now called with two arguments instead of one in certain conditions. This is consistent with the changes made to the `openChat` function definition. It's important to ensure that these changes do not introduce unexpected behavior in the application.
```diff
- return openChat('SampleChatScreen');
+ return openChat('SampleChatScreen', 'AnonymousChannelList');

- return openChat('SignedChatScreen');
+ return openChat('SignedChatScreen', 'SignedChannelList');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->